MIT-LICENSE

```diff
@@ -1,4 +1,6 @@
-Copyright (c) 2007-2016 Christian Neukirchen <purl.org/net/chneukirchen>
+The MIT License (MIT)
+
+Copyright (C) 2007-present Christian Neukirchen <http://chneukirchen.org/infopage.html>
```

Comment:
Unfortunately we need to be explicit about the year. So it should be 2007-2018

Reply:
Noted with thanks. I fixed it. be698cb